funnel/templates/project.html.jinja2

```diff
@@ -67,51 +67,89 @@
+        <h1 class="mui--text-display2"><strong>{{ project.title|e }}</strong></h1>
+        <p class="mui--text-headline">{{ project.tagline|safe }}</p>
+        <div class="profile-details">
+          {%- if project.profile.logo_url %}<img class="profile-details__logo" src="{{ project.profile.logo_url }}" alt="{{ project.profile.title }}"/>{% endif %}<span class="mui--text-subhead">by {{ project.profile.title|e }}</span>
+        </div>
+      </div>
+      <div class="grid__col-xs-12{% if project.banner_video_url or project.bg_image %} grid__col-md-5 grid__col-lg-4{% else %} push-date-wrapper{% endif %} date-wrapper xs-sm-padding">
+        <div>
+          <div class="date-icon">
+            <i class="material-icons mui--text-display1 mui--text-light">date_range</i>
+          </div>
+          <div class="date-details">
+            <p class="mui--text-subhead zero-bottom-margin date-details__label">Date</p>
+            <p class="mui--text-title mui--text-light">{{ project.datelocation }}</p>
+          </div>
+        </div>
+        <div>
+          <div class="date-icon">
+            <i class="material-icons mui--text-display1 mui--text-light">place</i>
+          </div>
+          <div class="date-details">
+            <p class="mui--text-subhead zero-bottom-margin date-details__label">Venue</p>
+            <p class="mui--text-title mui--text-light">{{ project.venues[0].title }}</p>
+          </div>
+        </div>
+        <div class="project-btn-wrapper">
+          <ul class="mui-list--inline project-btn-wrapper">
+            {%- if project.boxoffice_data %}
+              <li class="cta">
+                <a href="#tickets" class="mui-btn mui-btn--raised mui-btn--primary smooth-scroll">{% trans %}Participate{% endtrans %}</a>
+              </li>
+            {%- endif %}
+            {% if project.state.SUBMISSIONS -%}
+              <li class="mui--hidden-xs">
+                <a class="mui-btn mui-btn--raised mui-btn--accent" href="{{ project.url_for('new-proposal') }}">{% trans %}Propose a session{% endtrans %}</a>
+              </li>
+            {% endif %}
+            {%- if project.sessions %}
+              <li class="mui--hidden-xs mui--hidden-sm mui--hidden-md {% if project.state.SUBMISSIONS -%}float-btn{% endif %}">
+                <a href="{{ project.url_for('schedule') }}" class="mui-btn mui-btn--raised">{% trans %}View Schedule{% endtrans %}</a>
+              </li>
+            {%- endif %}
+          </ul>
+        </div>
       </div>
     </div>
   </div>
+{% endblock %}
 
+{% block contentwrapper %}
   <div class="grid details-section">
-    <div class="grid__col--bleed-y">
-      <hr class="mui-divider">
-    </div>
     <div class="grid__col-xs-12 grid__col-md-5 grid__col-lg-4">
-      <div class="sticky-admin-panel">
-        {%- if project.current_roles.admin or'checkin-event' in g.permissions %}
+      {%- if project.current_roles.admin or'checkin-event' in g.permissions %}
```

Comment:
Formatting issue: should be `or 'checkin-event'` (space missing). Also, use `current_auth.permissions.checkin-event` instead of `'checkin-event' in g.permissions`.

Reply:
Just realised `current_auth.permissions.checkin-event` won't work because of the hyphen. We have to update all permission names in the app to use underscores instead of hyphens.